docs/resources/client/level-components/inner-level-string.md

```diff
@@ -49,6 +49,25 @@ The following keys were valid prior to 2.0 and are deprecated, since they are in
 | kS36 | Color 4      | 4                |
 | kS37 | 3DL Color    | 1003             |
 
+***Pre-1.9 Keys***
+
+The following keys were valid prior to 1.9 and were deprecated as of 1.9, since they were included in the keys kS29, kS30, kS31, kS32, and kS33. Each color, instead of using one key, used 3 keys to represent Red, Green, and Blue values, respectively. In 1.7 and 1.8, the game used a fourth key to determine if a color channel used a player color, and which player color it would use.
```

Comment:
You still didn't change to `kS29`. Also, `kS29`-`kS33` should be just fine.

Reply:
I'm not sure what you mean by this. Should I remove "the keys?"

Reply:
I meant to format using \`kS29\`, just like how I've been doing in these reviews. The other comment refers to simplifying the range into "\`kS29\`-\`kS33\`".

Reply:
Ohhh I see. Ok